tests/unit/bokeh/document/test_document.py

```diff
@@ -873,45 +873,48 @@ def patch_test(new_value: Any):
         assert 'screen' == root1.foo_units
 
     def test_patch_reference_property(self) -> None:
-        d = document.Document()
-        assert not d.roots
-        assert len(d.models) == 0
```

Comment:
Is this necessary? There's only one important line change in this commit but that's really not immediately apparent from a page full of diff.